Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBDatabase-selectGridFSBucket.txt

MongoDB\\Database::selectGridFSBucket()
=======================================

.. meta::

>    :description: Select a GridFS bucket within a MongoDB database using the `selectGridFSBucket()` method, with customizable options for bucket name, chunk size, and more.

Reviewer: ```suggestion
   :description: Select a GridFS bucket in a MongoDB database with the MongoDB PHP Library, with customizable options for bucket name, chunk size, and more.
```
